app/FileTypes/JavaPropertiesFile.php

```diff
@@ -166,7 +166,7 @@ public function getLanguage() : string
 
     public static function getTypeName() : string
     {
-        return 'Java properties';
```

Comment:
https://en.wikipedia.org/wiki/.properties is not https://en.wikipedia.org/wiki/Java_(programming_language).

If you insist on changing this, perhaps '.properties (Java)' would work.